test/katex-spec.js

```diff
@@ -3066,6 +3066,21 @@ describe("A macro expander", function() {
             .toParseLike`11\sqrt[2]{2}11`;
     });
 
+    it("array cells generate groups", () => {
+        expect`\def\x{1}\begin{matrix}\x&\def\x{2}\x&\x\end{matrix}`
+            .toParseLike`\begin{matrix}1&2&1\end{matrix}`;
+    });
+
+    // TODO: This doesn't yet work; before the environment gets called,
```

Comment:
Could open an issue for this and include a link to the issue from this comment?

Reply:
Done.

Reply:
Thanks!